redcap_bridge/test_redcap/test_server_elab_interface.py

```diff
@@ -0,0 +1,41 @@
+from redcap_bridge.server_elab_interface import (create_template_without_metadata, create_template_with_metadata,
+                                                 create_template_with_converted_csv, download_experiment)
+from redcap_bridge.test_redcap.test_utils import (test_directory, initialize_test_dir)
+
+SERVER_CONFIG_YAML = (test_directory / 'testfiles' / 'TestProject' / 'project.json').resolve()
+
+test_file = (test_directory / 'testfiles' / 'elabConversion' / 'csvRecord.csv')
+
+
+def test_create_template_without_metadata(initialize_test_dir):
+    template_file = test_directory / 'testfiles' / 'elab_template.json'
+
+    res = create_template_without_metadata(server_config_json=SERVER_CONFIG_YAML, template_file=template_file)
+
+    assert res is not None
```

Comment:
I am not sure if testing for `None` covers the case in which the upload did not work. Are you sure this is not returning something else in that case?